Automattic/wp-calypso
# Copy Site: Include Marketplace Plugins when purchasing a new Copy Site

See https://github.com/Automattic/dotcom-forge/issues/1514

## Proposed Changes

Includes any Marketplace plugins on the origin site in the copy site checkout flow.

## Testing Instructions

1. Create a new Business site and take it Atomic.
2. Purchase WooCommerce Subscriptions.
3. Initiate 'Copy site' flow.
4. Verify WooCommerce Subscriptions is included in the checkout cart.
5. Purchase the site copy.
6. Verify WooCommerce Subscriptions add-on is included on your new site.
